deployment/aws-batch/job-definitions/production-pfb-analysis-run-job.json

```diff
@@ -0,0 +1,45 @@
+{
+    "jobDefinitionName": "production-pfb-analysis-run-job",
+    "type": "container",
+    "parameters": {},
+    "containerProperties": {
+        "image": "{image}",
+        "vcpus": 8,
```

Comment:
Yeah, I think that's a good idea.  Aside from the tiling job using that one CPU more efficiently, this would also reduce the possibility of a single tiling job tying up a whole instance (which I think is a possibility).